Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * ********Lesson Notes for me here********
+ * Requirements for this bootcamp lesson(lesson 3)
+ *  1. Need to have refactored code to recieve inputs
+ *  2. Need to gather the following pieces of information:
+ *      -Name(string}
+ *      -Age(int)
+ *      -Height(Float)[This will be in Meters]
+ *      -Weight(Float)[This will be in Killograms]
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void RunLesson()
+        {
+            string name;
+            int age;
+            float personHeight,
+                  personWeight,
+                  bmi;
```

Comment:
It is uncommon to declare variables at the top. This will force the reader to memorize the types of all variables. You should declare (and initialize) variables as late as possible (which is usually, but not always) when they are initialized.
This also let's the reader focus on what is happening in those lines instead of hovering/scrolling through your code.

Reply:
Changed this to the proper format as suggested here in the comments.